randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1608,40 +1607,62 @@ Extra - room_id: [35]
   > Door to Aquarium Speedway
       All of the following:
           Gravity Suit and Screw Attack
-          Any of the following:
-              Space Jump or Wall Jump (Advanced)
-              Hi-Jump and Wall Jump (Intermediate)
+          Space Jump or Wall Jump (Intermediate)
 
 > Other to Diffusion Connection Access; Heals? False
   * Layers: default
   * Tunnel to Diffusion Connection Access/Other to Powamp Shaft
   * Extra - door_idx: (112,)
-  > Door to Owtch Atrium
-      Missiles and Morph Ball
-  > Upper Ladder
-      Missiles ≥ 2 and Morph Ball
+  > Top Alcove
+      Trivial
 
 > Upper Ladder; Heals? False
   * Layers: default
   > Door to Owtch Atrium
       Missiles and Morph Ball
   > Door to Aquarium Speedway
       Trivial
+  > Top Alcove
+      Missiles and After Powamp Shaft Diffusion Blocks Destroyed
+  > Event - Break Missile Blocks from Below
+      All of the following:
+          Missiles ≥ 2
+          Any of the following:
+              Diffusion Missile Data
```

Comment:
Not objecting, but what was the reasoning for makinf diffusion require 2 missiles?

Reply:
this is due to the fact that you have to fire two missiles for the movement beginner method, although I guess you dont necessarily need to hard require 2 missiles all the time.

Reply:
yeah, with diffusion you'd only need one.
idk this seems like such a detail that itd only affect really... "special" presets, and those can just live with it. just wanted to know the reasoning since its not immediately clear.